File: eOftamoloskiCentar/eoftamoloskimobile/lib/model/korisnickiRacun.dart
import 'dart:ffi';

import 'package:json_annotation/json_annotation.dart';

part 'korisnickiRacun.g.dart';

@JsonSerializable()
class KorisnickiRacun {
  int? korisnickiRacunId;
  String? ime;
  String? prezime;
  String? email;
  String? brojTelefona;
  String? adresa;
  String? korisnickoIme;
  DateTime? datumRodjenja;

  KorisnickiRacun() {}

  //this is smth that enables for us to create actually some KorisnickiRacun
  factory KorisnickiRacun.fromJson(Map<String, dynamic> json) =>
      _$KorisnickiRacunFromJson(json);

  /// Connect the generated [_$KorisnickiRacunToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$KorisnickiRacunToJson(this);
}

File: eOftamoloskiCentar/eOftamoloskiCentar.Model/KorisnickiRacun.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Text;

namespace eOftamoloskiCentar.Model
{
    public class KorisnickiRacun
    {
        public int KorisnickiRacunId { get; set; }
        public string Ime { get; set; }
        public string Prezime { get; set; }
        public string Email { get; set; }
        public string BrojTelefona { get; set; }
        public string Adresa { get; set; }
        public DateTime DatumRodjenja { get; set; }
        public string KorisnickoIme { get; set; }
        public string LozinkaHash { get; set; }
        public string LozinkaSalt { get; set; }
        //public bool? Status { get; set; }


        [JsonIgnore]
        public string ImePrezime
        {
            get { return Ime + " " + Prezime; }
        }
    }
}
